client/finality-grandpa/src/until_imported.rs

```diff
@@ -70,13 +72,17 @@ pub(crate) trait BlockUntilImported<Block: BlockT>: Sized {
 }
 
 /// Buffering imported messages until blocks with given hashes are imported.
+#[pin_project::pin_project]
 pub(crate) struct UntilImported<Block: BlockT, BlockStatus, BlockSyncRequester, I, M: BlockUntilImported<Block>> {
-	import_notifications: Fuse<Box<dyn Stream<Item = BlockImportNotification<Block>, Error = ()> + Send>>,
+	#[pin]
```

Comment:
🤯

Reply:
It's unnecessary here though, because `UnboundedReceiver` is `Unpin` :sweat_smile: